www/src/utils/i18n.js

```diff
@@ -74,10 +74,14 @@ function getLocaleAndBasePath(path) {
   return { locale: defaultLang, basePath: path }
 }
 
+function makeDefaultLang() {}
+
 module.exports = {
   langCodes,
   langs,
   defaultLang,
+  isDefaultLang,
```

Comment:
don't worry about these two functions right now. Also "Default Lang" in this context is not what you think it is… it's the fallback lang that's rendered (which is always English).